tests/testthat/setup-workflow-ref.R

```diff
@@ -153,13 +153,14 @@ copy_all_output_dirs <- function() {
 create_rlg_models <- function() {
   # copy models before creating run log
   mod1 <- read_model(MOD1_PATH)
-  copy_model_from(mod1, basename(NEW_MOD2), NEW_DESC, .add_tags = NEW_TAGS)
-  copy_model_from(mod1,
-                  basename(NEW_MOD3),
-                  NEW_DESC,
-                  .based_on_additional = get_model_id(NEW_MOD2),
-                  .inherit_tags = TRUE,
-                  .update_model_file = FALSE)
+  copy_model_from(mod1, basename(NEW_MOD2), .add_tags = NEW_TAGS)
```

Comment:
removed the description from these two so that I could test having null descriptions in the run log